src/pages/task/taskTabs/logs/logTypes/TaskEventLogLine.tsx

```diff
@@ -105,12 +106,12 @@ export const TaskEventLogLine: React.FC<TaskEventLogEntry> = ({
 };
 
 const Row = styled.div`
-  font-size: 1.2em;
-  padding-top: 8px;
-  margin-bottom: 8px;
+  font-size: ${fontSize.l};
```

Comment:
The large font size here looks a bit comically large while medium looks too small... can we make it `16px` 😭